addons/cert.py

```diff
@@ -233,6 +233,16 @@ def int31(data, platform):
                     'Ensure that integer conversions do not result in lost or misinterpreted data (casting ' + str(value.intvalue) + ' to ' + destType + ')',
                     'INT31-c')
                 break
+
+
+# ENV33-C
+# Do not call system()
+def env33(data):
+    for token in data.tokenlist:
+        if isFunctionCall(token, ('system',), 1):
```

Comment:
can we remove the comma in `('system',)` or does that have some purpose?

Reply:
We need something iterable here: https://github.com/danmar/cppcheck/blob/f89b96145f1ac01b213590b6d38010fce38589f5/addons/cert.py#L90 I can't just send string, because in this case I will iterate over it characters.

Reply:
aha.. so ('system') is a string that is natural.